order/buyer/receive/simple-order-event.md

```diff
@@ -0,0 +1,167 @@
+* `ReadyToShip`: full quantity ready to be shipped by the supplier
+* `Shipped`: full quantity shipped by the supplier
+* `Delivered`: full quantity delivered at the buyer
+* `Cancelled`: cancelled by the buyer
+  {% endhint %}
+
+## Order lines
+
+`lines` contains one or more order lines:
+
+* `id`: the Tradecloud line identifier
+* `buyerLine`: the buyer part of the order line, see [Buyer line](#buyer-line).
+* `supplierLine`: the supplier part of the order line, see [Supplier line](#supplier-line).
+* `confirmedLine`: the order line as agreed between buyer and supplier. Advised is to use the `statusLine.deliverySchedule` and `statusLine.prices` instead.
+* `statusLine`: the order line values representing the current `Issued`, `Confirmed` or `InProgess` status, see [Status line](#status-line).
+* `indicators.deliveryOverdue` is true when the order line is overdue.
+* `status.processStatus`: the order line's [Process status](#process-status).
+* `status.logisticsStatus`: the order line's [Logistics status](#logistics-status).
+* `eventDates`: some key line event date/times
+* `mergedItemDetails`: detailed part information provided by both buyer and supplier, see [Item details](#item-details).
+* `lastUpdatedAt`: is the latest date time the order line has been changed.
+
+### Buyer line
+
+`buyerLine` is an echo of your order line fields as explained in [Issue a new order](../issue/#lines)
+
+### Supplier line
+
+`supplierLine` contains the supplier order line fields:
+
+* `salesOrderNumber`: the sales order number as known in the supplier's ERP system
+* `salesOrderPosition`: the position within the supplier's sales order
+* `description`: a free format additional description of this line by the supplier
+* `requests`: the supplier can request different delivery schedule, prices and charge lines. Advised is to use the `statusLine.deliveryScheduleInclRequests` and `statusLine.pricesInclRequests` instead.
+* `properties`: are key-value based custom fields, added by the supplier
+* `notes`: are simple custom fields, added by the supplier
+* `documents`: contain meta data, objectId or url, of attached documents by the supplier.
+
+{% page-ref page="download-document.md" %}
+
+### Status line
+
+The status line represents the current order line values related to the current `Issued`, `Confirmed` or `InProgress` status. The `InclRequests` fields include the `InProgress` status and open buyer or supplier request values.
+
+#### Simple delivery schedule
+
+When using `simpleOrderEvent` the simple delivery schedule is used:
+
+* `scheduledDelivery`: the current delivery line, either having `Issued` or `Confirmed` values.
+* `scheduledDeliveryInclRequests`: the current delivery line, either having `Issued`, `InProgress` requests or `Confirmed` values.
+
+##### Scheduled delivery fields
+
+* `date`: the delivery date of this delivery line. Date has ISO 8601 date `yyyy-MM-dd` format. See also [Standards](../../api/standards.md).
```

Comment:
maybe it's better to use full path here? I see you made it a "sub-header", but this isn't clearly rendered as such in the Gitbooks. It looks like it's on the same level as "Simple Delivery Schedule", which is confusing.

Reply:
I see the issue, but there are two paths `scheduledDelivery` and `scheduledDeliveryInclRequests`, thats why it is structured like this.

Reply:
Improved